### ISSUE TITLE
what is the best detection_eval in SynthText dataset

### ISSUE THREAD

**Issue by zjuDebug** [MhLiao/TextBoxes#7]
@MhLiao ,hello
I am training TextBoxes model to re-implementation the result in your paper on SynthText dataset.
So I have two questions:
   1. what is the best detection_eval in SynthText dataset in your training.
   2. did you chose 50k caffemodel to fine-tune with icdar2013 dataset or just chose a best
detection_eval (not 50k in my training)

Thanks in advance!

**Comment by MhLiao** [comment]
@zjuDebug I did not log the detection_eval because what I cared about is the F-measure.
By the way, there is a mistake in the previous codes of training, where the batch size should be 32, as the paper shows.

**Comment by zjuDebug** [comment]
@MhLiao Thanks a lot

**Comment by U019TT** [comment]
@MhLiao You just said you didn't care about the detection_eval value. But I wonder how you judge the network's performance on the validation data?

**Comment by MhLiao** [comment]
@enmengyi I judge the network's performance by the dection_eval value on the validation data of SynthText when I train the network on SynthText. But I did not log it or mentioned it in the paper because is final evaluation performance is F-measure. I have reimplemented it for several times. 50k iterations for batch size 32 is recommended.

**Comment by U019TT** [comment]
@MhLiao Thank you! 
That is, when the detection_eval basically no longer drops (assuming the learning rate has been set to small enough), the training can be over?
